Code/CSharpException/Triangle.cs

```diff
@@ -0,0 +1,37 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+
+namespace CSharpException
+{
+    public class Triangle
+    {
+        public Triangle(double sideA, double sideB, double sideC) 
+        {
+            if (sideA <= 0 || sideB <= 0 || sideC <= 0)
```

Comment:
better to create separate method to check the sides of the triangle.

Reply:
Done. Two separate methods are created.